uui-components/src/overlays/Dropdown.tsx

```diff
@@ -433,6 +435,20 @@ function DropdownComponent(props: DropdownProps, ref: React.ForwardedRef<HTMLEle
         updateTogglerSize();
     }
 
+    useEffect(() => {
+        if (!open) {
+            fixedPositionRef.current = null;
```

Comment:
I would suggest to rename fixedPositionRef to initialBodyPosition, to be more clear

Reply:
Done